client/src/document/index.scss

```diff
@@ -447,14 +447,11 @@ pre {
   padding: 0 1rem;
   position: relative;
 
-  &::after {
+  &::before {
```

Comment:
Could we add a margin-left, perhaps 8px, as the text can still get very close to the icon and look a bit weird:

![screenshot](https://user-images.githubusercontent.com/755354/228898199-2e659100-78f1-4d94-ad3f-1393db0c90f4.png)